apps/src/blockly/cdoBlocklyWrapper.js

```diff
@@ -274,6 +274,9 @@ function initializeBlocklyWrapper(blocklyInstance) {
     },
     getCode: function (workspace) {
       return Blockly.Xml.domToText(Blockly.Xml.blockSpaceToDom(workspace));
+    },
+    setStrictCheck: function (inputRow, type, strict) {
+      strict ? inputRow.setStrictCheck(type) : inputRow.setCheck(type);
```

Comment:
Seeing that `setStrictCheck` is defined as `setCheck(check, true)`, could you just call on `inputRow.setStrictCheck(type, strict)`?